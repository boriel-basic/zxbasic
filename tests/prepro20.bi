' Must return 
' PRINT );

#define macro(x) PRINT x;

macro())

