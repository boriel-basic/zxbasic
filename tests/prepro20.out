#line 1 "prepro20.bi"





PRINT ;)

